transcom/mymove
# B-20312 Update shipment pickup verbiage INT

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20312)

## Summary

Updated pickup verbiage and added some test coverage in.

To test:
1. Create a new move
2. When you pick a shipment type to create, select HHG.
3. Verify you see the pickup hint.
4. Back out and instead of HHG, select PPM.
5. Verify you do NOT see the pickup hint.
6. Back out and instead of PPM, pick NTS.
7. Verify you see the pickup hint.


<img width="619" alt="image" src="https://github.com/transcom/mymove/assets/147537467/88ce845c-d034-47d0-ab69-ad030d300727">


<img width="608" alt="image" src="https://github.com/transcom/mymove/assets/147537467/668d6931-e369-4713-8bf1-4775eb6cae49">
